build_debian.sh

```diff
@@ -387,9 +387,6 @@ set /files/etc/sysctl.conf/net.ipv6.conf.default.keep_addr_on_down 1
 set /files/etc/sysctl.conf/net.ipv6.conf.all.keep_addr_on_down 1
 set /files/etc/sysctl.conf/net.ipv6.conf.eth0.keep_addr_on_down 1
 
-set /files/etc/sysctl.conf/net.ipv6.conf.eth0.accept_ra_defrtr 0
-set /files/etc/sysctl.conf/net.ipv6.conf.eth0.accept_ra 0
```

Comment:
i prefer to keep it as it is. if you need dhcp, then you can enable it. I saw you have this in you interfaces.j2 file.

Reply:
If we have it in /etc/sysctl.conf, there can be a case where the setting done by interface.j2 is overwritten when sysctl files are re-read by a different service. I moved setting of accept_ra to a new file "files/dhcp/90-dhcp6-systcl.conf.j2". This will decide whether accept_ra is 1/0. For DHCPv6 on eth0 to work, accept_ra must be 1. 

The changes done as part of this PR enable DHCPv6 on eth0 by default just like how DHCP is enabled on eth0. This allows access to the switch using an IPv6 address. It has dual purpose to  be used for ZTP over IPv6 as well as using it for management of SONiC switch using DHCPv6 assigned IPv6 address.

If we set accept_ra to 0 by default, the access to the switch using DHCPv6 is not possible.

Following are some options for us to consider:

1. DHCPv6 is enabled when MGMT_INTERFACE table is not defined
>> This is part of the proposed change.

2. DHCPv6 is enabled only when ZTP discovery is being performed. 
>> This might momentarily work but user may lose connectivity to management interface if MGMT_INTRFACE configuration is not done or if accept_ra setting is not changed as part of ZTP configuration steps.

3. DHCPv6 is enabled only when ZTP is enabled in the build
>> This can be confusing as certain build combinations allows IPv6 management access out of the box and not in others.